frontend/public/stylesheets/content-styles.css

```diff
@@ -0,0 +1,304 @@
+/*
```

Comment:
FYI normally you don't want to copy their styles you probably want to grab it from node modules if its not being bundled in by default because if we update the package the styles will get out of date.